lib/BloqadeQMC/src/BloqadeQMC.jl

```diff
@@ -18,6 +18,9 @@ using LinearAlgebra
 
 using Random
 
+import BloqadeLattices: rydberg_interaction_matrix, AtomList
```

Comment:
can we avoid these `import`? they are bad. 

```suggestion
using BloqadeLattices: rydberg_interaction_matrix, AtomList
```

Reply:
Now I really want to understand when to use `import` haha

Reply:
NEVER ever use `import`, most people don't need it. It is one of those bad designs sneak through the Julia 1.0 feature freezing period.